src/resolver/resolver.zig

```diff
@@ -4060,43 +4055,6 @@ pub const Resolver = struct {
     }
 };
 
-pub const Dirname = struct {
-    pub fn dirname(path: string) string {
-        if (path.len == 0)
-            return std.fs.path.sep_str;
-
-        const root = brk: {
-            if (Environment.isWindows) {
-                const root = ResolvePath.windowsFilesystemRoot(path);
-                std.debug.assert(root.len > 0);
-                break :brk root;
-            }
-            break :brk "/";
-        };
-
-        var end_index: usize = path.len - 1;
-        while (bun.path.isSepAny(path[end_index])) {
-            if (end_index == 0)
-                return root;
-            end_index -= 1;
-        }
-
-        while (!bun.path.isSepAny(path[end_index])) {
-            if (end_index == 0)
-                return root;
-            end_index -= 1;
-        }
-
-        if (end_index == 0 and bun.path.isSepAny(path[0]))
-            return path[0..1];
-
-        if (end_index == 0)
-            return root;
-
-        return path[0 .. end_index + 1];
-    }
-};
```

Comment:
Removed the `Dirname.dirname` helper which seems to deviate [on the `null` case of `std.fs.path.dirname`](https://github.com/ziglang/zig/blob/92211135f1424aaca0de131cfe3646248730b1ca/lib/std/fs/path.zig#L805-L871).